manifests/plugin/gutterball.pp

```diff
@@ -1,9 +1,26 @@
 # gutterball plugin
 class katello::plugin::gutterball{
+  $foreman_gutterball_gem_config_tmp = '/tmp/foreman_gutterball.yaml'
+
+  $package = "${foreman::plugin_prefix}gutterball"
+  case $::osfamily {
+    'Debian': {
+      $gb_package_name = regsubst($package,'_','-','G')
+    }
+    default: {
+      $gb_package_name = $package
+    }
+  }
+
   Class[ 'certs' ] ->
   class { 'certs::gutterball': } ->
   foreman::plugin { 'gutterball': } ->
   class { '::gutterball':
     keystore_password => $certs::gutterball::gutterball_keystore_password,
+  } ~>
+  class { 'katello::plugin::gutterball::config':
```

Comment:
I think this doesn't have to be a separate class since it's only a single file. I even intend to make this possible in https://github.com/theforeman/puppet-foreman/pull/281 with a parameter to ```foreman::plugin```.